lightning-block-sync/src/init.rs

```diff
@@ -9,7 +9,7 @@ use lightning::chain;
 
 /// Returns a validated block header of the source's best chain tip.
 ///
-/// Upon success, the returned header can be used to initialize [`SpvClient`]. Useful during a fresh
+/// Upon success, the returned header can be used to initialize [`crate::SpvClient`]. Useful during a fresh
```

Comment:
I think this one was just missing the explicit link later. I don't think we want `crate::` rendered.

Reply:
Right, I didn't think it mattered much, happy to remove.